packages/datadog-plugin-sharedb/src/index.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+const MessagesAwaitingResponse = new WeakMap();
+const READABLE_ACTION_NAMES = {
+  hs: 'handshake',
+  qf: 'query-fetch',
+  qs: 'query-subscribe',
+  qu: 'query-unsubscribe',
+  bf: 'bulk-fetch',
+  bs: 'bulk-subscribe',
+  bu: 'bulk-unsubscribe',
+  f: 'fetch',
+  s: 'subscribe',
+  u: 'unsubscribe',
+  op: 'op',
+  nf: 'snapshot-fetch',
+  nt: 'snapshot-fetch-by-ts',
+  p: 'presence-broadcast',
+  pr: 'presence-request',
+  ps: 'presence-subscribe',
+  pu: 'presence-unsubscribe'
+};
+
+function createTraceName(action, collection) {
+  let actionName = READABLE_ACTION_NAMES[action];
+  if (actionName === undefined) {
+    actionName = action;
+  }
+  let traceName = 'sharedb-request/' + actionName;
+  if (collection) {
+    traceName += '/' + collection;
```

Comment:
Things like the collection name would go in a tag, and potentially also the `resource.name` tag if it's considered the main resource for this operation. Examples of typical resource names would be a web framework route, database queries, message queue topics, etc.

Reply:
Will change resource to simply be `sharedb` and use tags for the other information.

This is a change to how our plugin works internally... but I guess this is the convention other plugins follow. :)